modules/health_quest/app/services/health_quest/pgd_service.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+# rubocop:disable all
```

Comment:
Is this a temporary move?

Reply:
Sorry, what happened is I had a PR that failed for no reason because danger/danger crashed so I redid a smaller PR and it looks like I pulled in some code that had the wrong rubocop disable